subgraph/tests/common.ts

```diff
@@ -14,11 +14,9 @@ export function mockPollContract(): void {
     ethereum.Value.fromI32(20),
   ]);
 
-  createMockedFunction(DEFAULT_POLL_ADDRESS, "treeDepths", "treeDepths():(uint8,uint8,uint8,uint8)").returns([
+  createMockedFunction(DEFAULT_POLL_ADDRESS, "treeDepths", "treeDepths():(uint8,uint8)").returns([
```

Comment:
we should also remove all unnecessary code from the Poll subgraph now that there's no more mergeMessageAq events

Reply:
resolved